.github/workflows/test.yaml

```diff
@@ -0,0 +1,92 @@
+name: test
+
+on:
+  push:
+    branches:
+      - main
+      - stable
+      - "*.*-maintenance"
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          version: 9.15.1+sha512.1acb565e6193efbebda772702950469150cf12bcc764262e7587e71d19dc98a423dff9536e57ea44c49bdf790ff694e83c27be5faa23d67e0c033b583be4bfcf
+          run_install: false
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 23
+          cache: pnpm
+      - run: pnpm install
+      - run: pnpm run check:ci
+
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:16-alpine
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_DB: hollo_test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10ms
+          --health-timeout 3s
+          --health-retries 50
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:7-alpine
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10ms
+          --health-timeout 3s
+          --health-retries 50
+        ports:
+          - 6379:6379
+    env:
+      # HOME_URL=https://localhost/  # optional; if present, the home page will redirect to this URL
+      DATABASE_URL: postgresql://postgres:postgres@localhost:5432/hollo_test
+      REDIS_URL: redis://localhost/1
```

Comment:
I think we don't need Redis anymore!

Reply:
Oh? Is queues now just PostgreSQL? What about session cookies?

Reply:
Session cookies are built on HMAC-based secure cookies.

Reply:
```suggestion
```